### PR TITLE
added group list, icon, and links

### PR DESCRIPTION
**Pull request by FlyingUnicornSF**
added group list, icon, and links that are associated with the narrative on share panel.

### CODE REVIEW

File: kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSharePanel.js [hunk 1]
```diff
@@ -92,19 +101,59 @@ define ([
...
         /**
-         * Update global variable, orgList, then call render function.
-         * @param {array} list
+         * fetch organization info from id
+         * extract logo url and map it to org id with org name.
+         * @param {string} org org id
+         * @param {string} token auth token
+         * @param {Map} map empty map
+         */
+        fetchOrgLogoUlr: function(org, token, map) {
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSharePanel.js]: fetchOrgLogoUrl? or Ulr?

**Comment by FlyingUnicornSF** [comment]
@briehl @eapearson @jayrbolton 🙏

File: kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSharePanel.js [hunk 2]
```diff
@@ -92,19 +101,59 @@ define ([
...
+                json: true,
+                headers:{
+                    "Authorization": token,
+                    "Content-Type": "application/json",
+                },
+            })
+            .then(response => response.json())
+            .then(response => {
+                let orgInfoMap = new Map();
+                return Promise.all(response.map((org) => this.fetchOrgLogoUlr(org, token, orgInfoMap)));
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSharePanel.js]: fetchOrgLogoUrl*

File: kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSharePanel.js [hunk 3]
```diff
@@ -313,7 +362,21 @@ define ([
...
+                // if there are orgs already associated with the narrative, add the org list.
+                if(this.narrativeOrgList){
+                    var $narrativeOrgsDiv = $('<table>').css({'border': '1px solid rgb(170, 170, 170)', 'border-radius': '4px', 'text-align': 'left', 'width': '51%', 'padding': '10px', 'margin': 'auto', 'margin-top': '10px'});                    
+                    this.narrativeOrgList.forEach((value, key, map) => {
+                        let url = window.location.origin + "/#org/" + key;
+                        let $href = $('<a>').attr("href", url);
+                        let $logo = $('<img>').attr("src", value[1]).css({'width': '40', 'margin': '8px'});
+                        $href.append($logo).append(value[0]);
+                        let $tr = $('<tr>').css({'padding': '2px'}).append($href);
+                        $narrativeOrgsDiv.append($tr)
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSharePanel.js]: semicolon

**Pull request merged**